client/COPY.json

```
@@ -1267,6 +1267,19 @@
   "VHA_COMPLETE_TASK_MODAL_BODY": "Provide details such as file structure or file path",
   "VHA_COMPLETE_TASK_CONFIRMATION_PO": "Appeal has been marked as ready for review and sent to the CAMO team for review.",
   "VHA_COMPLETE_TASK_CONFIRMATION_VISN": "Appeal has been marked as ready for review and sent to the Program Office team for review.",
+  "VHA_RETURN_TO_BOARD_INTAKE_MODAL_TITLE": "Return to Board Intake",
+  "VHA_RETURN_TO_BOARD_INTAKE_MODAL_DETAIL": "Why is this appeal being returned?",
+  "VHA_RETURN_TO_BOARD_INTAKE_MODAL_BODY": "This appeal will be sent to Board Intake.",
+  "VHA_RETURN_TO_BOARD_INTAKE_DUPLICATE": "Duplicate",
+  "VHA_RETURN_TO_BOARD_INTAKE_HLR_PENDING": "HLR Pending",
+  "VHA_RETURN_TO_BOARD_INTAKE_SC_PENDING": "SC Pending",
+  "VHA_RETURN_TO_BOARD_INTAKE_NOT_VHA_RELATED": "Appeal is not related to a VHA issue",
+  "VHA_RETURN_TO_BOARD_INTAKE_CLARIFICATION_NEEDED": "Clarification needed from appellant",
+  "VHA_RETURN_TO_BOARD_INTAKE_NO_VHA_DECISION": "A VHA decision has not been made",
+  "VHA_RETURN_TO_BOARD_INTAKE_OTHER": "Other",
```

```suggestion
  "VHA_RETURN_TO_BOARD_INTAKE_NOT_VHA_RELATED": "Not VHA related",
  "VHA_RETURN_TO_BOARD_INTAKE_CLARIFICATION_NEEDED": "Clarification needed from appellant",
  "VHA_RETURN_TO_BOARD_INTAKE_NO_VHA_DECISION": "No VHA decision",
  "VHA_RETURN_TO_BOARD_INTAKE_OTHER": "Other",
```

Apologies! I did not attach the designs to this story. Here they are: https://bipuiux.invisionapp.com/console/share/ESDZU25T4VH/975710431/play